From 693215fda8d6f50721036964eb041fb80ad9a6c4 Mon Sep 17 00:00:00 2001
From: Wu Liangqing <wlq@rock-chips.com>
Date: Tue, 3 Jul 2018 09:52:53 +0800
Subject: [PATCH] arm64: dts: rockchip: px30-evb-ddr3-v10: vcc_3v0
 regulator-on-in-suspend

According to the hardware design, wifi and rmii cannot be powered off in
deepsleep

Change-Id: Id44ef9b31c34b6f12695dbf3fbb843950602ca71
Signed-off-by: Wu Liangqing <wlq@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/px30-evb-ddr3-v10.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/px30-evb-ddr3-v10.dts b/arch/arm64/boot/dts/rockchip/px30-evb-ddr3-v10.dts
index 2b34f7a39cfb..17b6a0cd9dba 100644
--- a/arch/arm64/boot/dts/rockchip/px30-evb-ddr3-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/px30-evb-ddr3-v10.dts
@@ -438,7 +438,7 @@
 				regulator-initial-mode = <0x2>;
 				regulator-name = "vcc_3v0";
 				regulator-state-mem {
-					regulator-off-in-suspend;
+					regulator-on-in-suspend;
 					regulator-suspend-microvolt = <3000000>;
 				};
 			};
-- 
2.35.3

